src/Components/Buttons/index.js

```diff
@@ -0,0 +1,116 @@
+import React, {Component} from 'react'
+import './index.css' 
+
+export class Single extends Component {
+
+    state = {
+        activeButton: ''
+    }
+
+    toggleClick = (title) => {
+        this.setState({
+            activeButton: title
+        })
+    }
+
+    renderButtons = ({items, active}) => { 
+        return (
+        items.map((button) => {
+            let title = button.title;
+            return (
+            <button
+                key={title}
+                className={active === title ? 'active' : ''}
+                onClick={() => this.toggleClick(title)}
+                title={title}
+            >
+                {title}
+            </button>
+            )
+        })
+        )
+    }
+
+    render() {
+        const active = this.state.activeButton;
+        return (
+            <div className='wrapper'>
+                <h1>Single</h1>
+                <div className='buttons'>
+                    <this.renderButtons items={this.props.items} active={active}/>
+                </div>
+                <Status status={this.state.activeButton}/>
+            </div>
+        )
+    }
+}
+
+export class Multiple extends Component {
```

Comment:
move to another file and folder
also folders should have the same name as component's